src/main/java/uk/gov/hmcts/reform/professionalapi/domain/Jurisdiction.java

```diff
@@ -11,3 +12,4 @@
 
     private String id;
 }
+*/
```

Comment:
Please delete this file rather than entirely comment it out

Reply:
Done